src/main/java/edu/greenblitz/tobyDetermined/commands/Auto/PreAutoCommand.java

```diff
@@ -0,0 +1,53 @@
+package edu.greenblitz.tobyDetermined.commands.Auto;
+
+import com.pathplanner.lib.PathPlannerTrajectory;
+import edu.greenblitz.tobyDetermined.commands.swerve.SwerveCommand;
+import edu.greenblitz.tobyDetermined.subsystems.swerve.SwerveChassis;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class PreAutoCommand extends SwerveCommand {
```

Comment:
Again with the Control + Alt + L.
We can probably create automation for this if you want.